db/v-2/tables/test-suites-source.xml

```diff
@@ -31,4 +31,12 @@
         <addUniqueConstraint tableName="test_suites_source" columnNames="organization_id,name" constraintName="test_suites_source_name"/>
         <addUniqueConstraint tableName="test_suites_source" columnNames="git_id,test_root_path,branch" constraintName="test_suites_source_git_location"/>
     </changeSet>
+
+    <changeSet id="add-latest-version" author="sanyavertolet">
```

Comment:
as I got, we introduce a new field just to support filtering `Show only latest version`. 
Is this a specific field required? Can we have an extra endpoint for it?

Reply:
It is going to be either null, or prefix of commit hash (just like GitHub does)

There will be an extra endpoint, but not sure it should be in this PR.

This field is filled with the first fetch (which mostly will accrue right after source is created)